docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,143 @@
+   +----+------------+---------------+---------------------------+---------------+
+   | ID | first_name | last_name     | user_email                | display_name  |
+   +----+------------+---------------+---------------------------+---------------+
+   |  1 | admin      | admin         | admin@yourcompany.com     | admin         |
+   |  2 | john       | doe           | john.doe@gmail.com        | john          |
+   |  3 | jane       | doe           | janedoe@ymail.com         | jane          |
+   +----+------------+---------------+---------------------------+---------------+
+   3 rows in set (0.00 sec)
+   ```
+
+1. Change the fields where PII Data is contained with [`UPDATE` Queries](https://dev.mysql.com/doc/refman/8.0/en/update.html).
+   For example, changing the first name of an user, when the email address is not using the company domain, would result in the following query:
+
+   ```sql
+   UPDATE user
+   SET first_name='redacted'
+   WHERE email NOT LIKE '%@yourcompany%'
+   ```
+
+   Adapt and run that query for all fields that need to be sanitized.
+   If you modified fields that shouldn't have been altered, [you can restore them from the previously taken dump](../administration/backup-and-restore.md#restore).
+
+   You can create a script to automate the sanitization process to be able to run it in [your deployment hook](../user_guide/reference/platform-app-yaml.html#hooks):
+
+    ```yaml
+    deploy: |
+      cd /app/public
+      if [ $PLATFORM_ENVIRONMENT = "main" ]; then
+        # Do whatever you want on the production site.
+      else
+        # The sanitization of the database should happen here (since it's non-production)
+      fi
+    ```
+
+<--->
+
+---
+title=When using Drupal
+file=none
+highlight=false
+---
+
+1. Take a [database dump](../configuration/services/mysql/_index.md#exporting-data). In this guide production data won't be altered, consider it merely as a safety precaution. Run `platform db:dump -e staging` to get a database dump of your staging environment,
+1. With Drupal, you can use [the `drush sql-sanitize` command](https://www.drupal.org/project/database_sanitize) to sanitize your database and get rid of sensitive, live, information.
+Add the sanitization of the database script in [your hook](../user_guide/reference/platform-app-yaml.html#hooks) for the non-production environment:
+
+  ```yaml
+  type: php:7.0
+  build:
+  flavor: drupal
+  hooks:
+  build: |
+    # Whatever you want to do during the build phase.
+  deploy: |
+    cd /app/public
+    if [ $PLATFORM_ENVIRONMENT = "main" ]; then
+      # Do whatever you want on the production site.
+    else
+      drush -y sql-sanitize --sanitize-email=user_%uid@example.com --sanitize-password=custompassword
+    fi
+    drush -y updatedb
+  ```
+
+{{< /codetabs >}}
+
+## What's next
+
+In this guide you learned how to remove sensitive data from a database.
+
+To keep meaningful data, you could add a `faker` in the process.
```

Comment:
```suggestion
To keep meaningful data, you could add a `faker` to the process.
```

I'd try to avoid "keep" because you don't want to keep the data that exists. You're trying to replace it with other data.